mhkit/wave/io/cdip.py

```diff
@@ -335,9 +339,11 @@ def request_parse_workflow(
                 all_2D_variables=all_2D_variables,
                 silent=silent,
             )
-            handle_caching(hash_params, cache_dir, data=data)
-        else:
-            data = data[0]
+            handle_caching(
+                hash_params,
+                cache_dir,
+                cache_content={"data": data, "metadata": None, "write_json": None},
+            )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args